apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -253,6 +262,23 @@ type ExtensionFunctions = {
   functions: AbiFunction[];
 };
 
+const FunctionInputSearch = ({
+  searchFn,
+}: {
+  searchFn: DebouncedState<(e: ChangeEvent<HTMLInputElement>) => void>;
+}) => (
+  <div className="sticky top-0 z-10 mb-3">
+    <div className="relative w-full">
+      <SearchIcon className="-translate-y-1/2 absolute top-[50%] left-3 size-4 text-muted-foreground" />
+      <Input
+        placeholder="Search"
+        className="rounded-none border-r-none border-l-none py-2 pl-9 focus-visible:ring-0"
+        onChange={searchFn}
+      />
+    </div>
+  </div>
+);
```

Comment:
Remove margin bottom on the search input - it adds unnecessary extra spacing 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/fd5aa9fa-5464-4a18-8028-a798548890ea.png)